### PR TITLE
New package: signal-backup-decode-0.1.5

### PR DESCRIPTION
**Pull request by sburris0**
#23529

### CODE REVIEW

**Comment by sburris0** [comment]
Is this a problem caused by my template or rust libraries not being buildable on i686/aarch64?

**Comment by ericonr** [comment]
At least the `i686` failure was because of connection issues. You could try to trigger the build again by reabsing and force pushing the PR branch.

**Comment by sburris0** [comment]
Should I just exclude aarch64?

**Comment by sgn** [comment]
https://github.com/DaGenix/rust-crypto/issues/383

**Comment by sburris0** [comment]
I've just asked the author to switch to the rust crypto library that is actually maintained still

File: srcpkgs/signal-backup-decode/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'signal-backup-decode'
+pkgname=signal-backup-decode
+version=0.1.5
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="libressl-devel"
+short_desc="Decode Signal Backups"
```
[the remark this answers is not in the document]
Comment (sburris0) [reply srcpkgs/signal-backup-decode/template:8]: I think it's implied. If I explicitly mention Signal Android then I might as well list the other platforms it supports too, which I don't see a reason for.

[the remark this answers is not in the document]
Comment (Piraty) [reply srcpkgs/signal-backup-decode/template:8]: It wasn't implied for me, as
1. I use signal on other platforms than Android
2. The projects description says: `This repository contains a small programm to decode a backup produced by Signal Android.` which (to me) sounds like it's limited to Android

**Comment by Piraty** [comment]
follow up: https://github.com/void-linux/void-packages/pull/25344

next time amend to existing PR instead of opening a new one, thanks!

**Pull request closed without merging**